### PR TITLE
Update getting-started.md to use .NET 5

### LINKED ISSUE #25098: Painless Upgrade to .NET 5
Hi

i just followed along and everything worked with no problem, using only port 80 but targeting .net 5.
You can check the code here if you need it: https://github.com/MACEL94/DaprMultiContainer

I'd say you can update guidelines to .NET 5, or i can submit a pull request if you want.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 83e95a3f-447b-8ca0-beb5-f0363896fe29
* Version Independent ID: b24e3ee5-404c-98a2-042e-6ac2d363b842
* Content: [Get started with Dapr](https://docs.microsoft.com/en-us/dotnet/architecture/dapr-for-net-developers/getting-started)
* Content Source: [docs/architecture/dapr-for-net-developers/getting-started.md](https://github.com/dotnet/docs/blob/main/docs/architecture/dapr-for-net-developers/getting-started.md)
* Product: **dotnet-architecture**
* GitHub Login: @amolenk
* Microsoft Alias: **nanil**

### PR DESCRIPTION
**Pull request by macel94**
Removed EXPOSE 443 in yaml that was unneccessary.
Used .NET 5 links

Update descriptions as .NET Core now is .NET(also the visual studio workload)

## Summary

Describe your changes here.

Fixes #25098

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25117) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25117) before we can accept your contribution.<br/><br/>:x: MACEL94 [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25117)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=25117) it.</sub>

**Comment by MACEL94** [comment]
#25098

File: docs/architecture/dapr-for-net-developers/getting-started.md [hunk 1]
```diff
@@ -371,34 +371,33 @@ In the final part of this example, you'll add container support and run the solu
     The *Dockerfile* contains the YAML:
 
     ```yml
-    FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+    FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
     WORKDIR /app
     EXPOSE 80
-    EXPOSE 443
```
Comment (amolenk) [inline docs/architecture/dapr-for-net-developers/getting-started.md:377]: Leave the 443 port in for the front-end. If you follow the instructions closely, you only disable HTTPS for the back-end project. The Docker file that VS generates for the front-end will contain both 80/443 ports.

File: docs/architecture/dapr-for-net-developers/getting-started.md [hunk 2]
```diff
@@ -371,34 +371,33 @@ In the final part of this example, you'll add container support and run the solu
...
     ```
-
+    
     The preceding *Dockerfile* sequentially performs the following steps when invoked:
-
-    1. Pulls the `mcr.microsoft.com/dotnet/aspnet:3.1` image and names it `base`.
+    
+    1. Pulls the `mcr.microsoft.com/dotnet/aspnet:5.0` image and names it `base`.
     1. Sets the working directory to */app*.
-    1. Exposes port `80` and `443`.
```
Comment (amolenk) [inline docs/architecture/dapr-for-net-developers/getting-started.md:400]: See previous comment; include 443.

**Review by amolenk — CHANGES_REQUESTED** [review]
Thanks for the PR! 

I've only got comments on the 443 port in the front-end.

@IEvangelist Can you review as well? 🙏

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**